apps/backend/config-server/src/main/resources/config/gateway/gateway-dev.yaml

```
@@ -12,3 +12,10 @@ spring:
     url: ${USER_DEV_DB_URL}
     username: ${POSTGRES_USERNAME}
     password: ${POSTGRES_PASSWORD}
+jwt:
+  secret: ${JWT_SECRET}
+app:
+  permitted:
+    routes: /api/v1/user/login,/api/v1/user/register,/api/v1/user/verify,/csrf,/test-csrf,/actuator/*
+  access-token-expiration: 3600
+  refresh-token-expiration: 86400
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)